server/command.go

```diff
@@ -246,27 +453,34 @@ func (p *Plugin) subscribeCommand(parameters []string, channelID string, config
 		return subscribeErr.Error()
 	}
 	var hookStatusMessage string
-	hasHook, err := p.HasProjectHook(info, namespace, project)
-	if err == nil {
-		if hasHook {
-			//web hook found
-			return fmt.Sprintf("Successfully subscribed to %s.", fullPath)
-		}
-		//no web hook found
-		hookStatusMessage = fmt.Sprintf(
-			"Please [setup a WebHook](%s/%s/%s/hooks) in GitLab to complete integration. See [setup instructions](%s) for more info.",
-			config.GitlabURL,
-			namespace,
-			project,
-			webhookHowToURL,
-		)
+	var hasHook bool
+	if project != "" {
+		hasHook, err = p.HasProjectHook(info, namespace, project)
+		if err != nil {
+			return fmt.Sprintf(
+				"Unable to determine status of Webhook. See [setup instructions](%s) to validate.",
+				webhookHowToURL,
+			)
+		}
 	} else {
-		//unable to get web hook info
-		hookStatusMessage = fmt.Sprintf(
-			"Unable to determine status of Webhook. See [setup instructions](%s) to validate.",
-			webhookHowToURL,
-		)
+		hasHook, err = p.HasGroupHook(info, namespace)
+		if err != nil {
+			return fmt.Sprintf(
+				"Unable to determine status of Webhook. See [setup instructions](%s) to validate.",
+				webhookHowToURL,
+			)
+		}
+	}
+
+	if hasHook {
+		//web hook found
+		return fmt.Sprintf("Successfully subscribed to %s.", fullPath)
 	}
+	//no web hook found
+	hookStatusMessage = fmt.Sprintf(
+		"\nA Webhook is needed, run ```/gitlab webhook add %s``` to create one now.",
+		fullPath,
+	)
 
-	return fmt.Sprintf("Successfully subscribed to %s. %s", fullPath, hookStatusMessage)
+	return fmt.Sprintf("Successfully subscribed to %s.%s", fullPath, hookStatusMessage)
```

Comment:
Nit pick: I would be nice if the message `Successfully subscribed to` would not have to be duplicated in the code.